docs/snap-clone.md

```diff
@@ -1,12 +1,18 @@
-# Create CephFS snapshot and Clone Volume
+# Create snapshot and Clone Volume
 
+- [Prerequisite](#prerequisite)
 - [Create CephFS snapshot and Clone Volume](#create-cephfs-snapshot-and-clone-volume)
-  - [Prerequisite](#prerequisite)
   - [Snapshot](#snapshot)
     - [Create snapshotclass](#create-snapshotclass)
     - [Create snapshot](#create-snapshot)
   - [Restore snapshot to a new PVC](#restore-snapshot-to-a-new-pvc)
   - [Clone PVC](#clone-pvc)
+- [Create RBD snapshot and Clone Volume](#create-rbd-snapshot-and-clone-volume)
```

Comment:
keep the table content same for both cephfs and rbd, currently it looks different

Reply:
okay.